### PR TITLE
kelly/update_terms_and_conditions

### PR DESCRIPTION
**Pull request by kellybinary**

### CODE REVIEW

File: src/templates/static/legal/tac_sections/affiliate.jsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ const Affiliate = () => (
...
+                    <li>{it.L('In the event that the Company is aware of an Affiliate who is in breach of any of the above provisions, the Company shall have the right, in addition to any other right or remedy available to it under this Agreement or applicable law, to immediately block the Affiliate\'s access to this Programme. The Affiliate hereby irrevocably waives its rights to, and shall indemnify the Company and any member of the Binary group of companies for, any claim or demand made against the Company or any member of the Binary group of companies, their directors, officers, shareholders, employees or against the [_1] website in respect of such action taken by the Company.', it.website_name)}</li>
```
Comment (nara-binary) [inline src/templates/static/legal/tac_sections/affiliate.jsx]: Suggest to change the wordings "any claim or demand made" to "any claims or demands made".

Comment (kellybinary) [reply src/templates/static/legal/tac_sections/affiliate.jsx]: ok, sure

File: src/templates/static/legal/metatrader/terms_and_conditions.jsx [hunk 2]
```diff
@@ -36,19 +35,18 @@ const TermsAndConditions = () => (
...
-                            <li>{it.L('You have sufficient experience and knowledge in matters of financial trading to be capable of evaluating the merits and risks of acquiring financial contracts via this site and have done so without relying on any information contained in this site.')}</li>
+                            <li>{it.L('You are over 18 years of age, unless you are an Estonian resident whereby you would have to be over 21;')}</li>
+                            <li>{it.L('You are not resident in a restricted country such as Costa Rica, Hong Kong, Israel, Jersey, Malaysia, Malta, USA or any other restricted country which has been identified by the FATF as having strategic deficiencies;')}</li>
+                            <li>{it.L('You have sufficient experience and knowledge in matters of contract for difference to be capable of evaluating the merits and risks of acquiring contracts via this site and have done so without relying on any information contained in this site.')}</li>
```
Comment (jy-binary) [inline src/templates/static/legal/metatrader/terms_and_conditions.jsx]: should be "contracts for difference"

File: src/templates/static/legal/metatrader/terms_and_conditions.jsx [hunk 3]
```diff
@@ -155,20 +159,34 @@ const TermsAndConditions = () => (
...
+                    <li>{it.L('The Stop Out Level applicable to your MT5 Real Account may differ from the default Stop Out Level published on our website.')}</li>
+                </ol>
+            </li>
+            <li>{it.L('Negative Balance Protection')}
+                <ol>
+                    <li>{it.L('The Company has a Stop Out policy which prevents you from losing more than you have deposited. However, in the event that your Open Positions are not closed when the Margin Level for your MT5 Real Account reaches the Stop Out Level, the Company, at its sole discretion, may waive your negative balance by crediting your account if your Account Balance goes into a negative balance. The negative balance is determined by aggregating all the client’s negative balances incurred over a 24-hour period across all accounts held by you. You acknowledge that the offer of the Negative Balance protection by the Company to you is at the Company’s sole discretion and we reserve the right to change the features and eligibility criteria of the Negative Balance Protection at any time. The provisions of the Negative Balance Protection do not apply:')}
```
Comment (jy-binary) [inline src/templates/static/legal/metatrader/terms_and_conditions.jsx]: This sentence "The negative balance is determined by aggregating all the client’s negative balances incurred over a 24-hour period across all accounts held by you" needs rewording. Who is "the client"? Is it "you"? Please ask Reuben

File: src/templates/static/legal/tac_sections/tac.jsx [hunk 4]
```diff
@@ -36,7 +36,7 @@ const Tac = () => (
...
+        <p>{it.L('Clients agree to be fully and personally liable for the due settlement of every transaction entered into under their account with the Company. This includes any transactions entered into by members of the Client\'s family or entourage who have gained access to the account. Clients are responsible for ensuring that they alone control access to the account, and that no minors are granted access to trading on the website. In any case, Clients remain fully liable for any and all positions traded on their account, and for any credit card transactions entered into the site for their account. Each Client also indemnifies the Company in respect to all costs and losses of any kind, whatsoever as may be incurred by the Company as a result, direct or indirect, of the Client\'s failure to perform or settle such a transaction. Online Gambling debts are enforceable in law in the Isle of Man.')}</p>
```
Comment (jy-binary) [inline src/templates/static/legal/tac_sections/tac.jsx]: the sentence "Online Gambling debts are enforceable in law in the Isle of Man." is coming out of no-where. Please discuss with Joanna. Either it should be removed, or if it needs to stay, then it needs context (i.e. "Clients who have accounts with BIOM.....")

**Pull request merged**